parity/cli/mod.rs

```diff
@@ -315,7 +315,7 @@ usage! {
 
 			ARG arg_ports_shift: (u16) = 0u16, or |c: &Config| c.misc.as_ref()?.ports_shift,
 			"--ports-shift=[SHIFT]",
-			"Add SHIFT to all port numbers Parity is listening on. Includes network port and all servers (RPC, WebSockets, UI, IPFS, SecretStore).",
+			"Add SHIFT to all port numbers Parity is listening on. Includes network port and all servers (HTTP JSON-RPC, WebSockets JSON-RPC, IPFS, SecretStore).",
```

Comment:
s/port/ports/ (plural s)

Also, not sure what is meant by "all servers" – perhaps "services" is better? That is, how is "notwork ports" different from other ports?

Maybe this: "Add SHIFT to all network port numbers used by Parity. This includes ports used by any enabled services (JSON-RPC over HTTP/WS, IPFS and SecretStore)." ?

Reply:
the plural is on the "number***s**", I think it's correct.
Network as in *network discovery* (30303 per default) parity run http and ws servers to listen to incoming calls, I think "server" is fine here.